src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img class="img-heder" 
+        src="images/logo.png" 
+        alt="logo"
+      >
+      <nav>
+        <ul class="strong ul_link">
+          <li class="navigation navigation_mar">
+            <a class="is-active nav_link" href="#Apple">Apple</a>
+          </li>
+
+          <li class="navigation ">
+            <a class="nav_link" href="#Samsung">Samsung</a>
+          </li>
+
+          <li class="navigation">
+            <a class="nav_link" href="#Smartphones">Smartphones</a>
+          </li>
+
+          <li class="navigation ">
+            <a
+              class="nav_link"
+              data-qa="hover"
+              href="#Laptops & Computers">Laptops & Computers
+            </a>
+          </li>
+
+          <li  class="navigation">
+            <a class="nav_link" href="#Gadgets">Gadgets</a>
+          </li>
+
+          <li class="navigation">
+            <a class="nav_link" href="#Tablets">Tablets</a>
+          </li>
+
+          <li class="navigation">
+            <a class="nav_link" href="#Photo">Photo</a>
+          </li>
+
+          <li class="navigation">
+            <a class="nav_link" href="#Video">Video</a>
+          </li>
+        </ul>
```

Comment:
use BEM please {block}__{element}
nav should have class nav, because it's navigation block
`<li>` inside should have class 'nav__item' for example :)
Fix everywhere